__fixtures__/root.scss

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2018 American Express Travel Related Services Company, Inc.
```

Comment:
not sure if this date matters?

Reply:
@nellyk happy to change it, do we change it on all files or just the new ones I created?

Reply:
Just new ones

Reply:
fixed on https://github.com/americanexpress/purgecss-loader/pull/9/commits/bddff983e899cfac17c8239e690a15db6a33351b